compiler/cgen.nim

```diff
@@ -860,7 +860,7 @@ proc containsResult(n: PNode): bool =
     for i in 0..<n.safeLen:
       if containsResult(n[i]): return true
 
-const harmless = {nkConstSection, nkTypeSection, nkEmpty, nkCommentStmt, nkTemplateDef, nkMacroDef} +
+const harmless = {nkConstSection, nkTypeSection, nkEmpty, nkCommentStmt, nkTemplateDef, nkMacroDef, nkMixinStmt} +
```

Comment:
shouldn't `nkMixinStmt` be stripped away in semantic pass before it reaches backend, eg, so that it doesn't affect each backend

Reply:
Maybe, but I'm generally in favor of not losing information in the AST. As you can see there are always things that backends explicitly need to ignore anyway.